klayout/drc/README.md

```diff
@@ -0,0 +1,93 @@
+# DRC Documentation
+
+Explains how to use the runset.
+
+## Folder Structure
+
+```text
+📦drc
+ ┣ 📦testing
+ ┣ 📦rule_decks
+ ┣ 📜README.md
+ ┗ 📜run_drc.py
+ ```
+
+## Rule Deck Usage
+The `run_drc.py` script takes a gds file to run DRC rule decks of GF180 technology with switches to select subsets of all checks. 
+
+### Requirements
+Please make sure to use the latest Klayout setup at your side.
+
+Also, please make sure to install the required python packages at `requirements.txt` by using
+```bash
+pip install -r requirements.test.txt
+```
+
+
+### Switches
+The list of switches used for running DRC:
```

Comment:
if those are option maybe we should also add the actual way to specificy those on the commandline?

Reply:
It's documented at the bottom how to use them from command line. I have removed that section anyway. 
Done